Sprint-3/quote-generator/index.html

```diff
@@ -3,13 +3,24 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Quote Generator App</title>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Italiana&family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
     <script defer src="quotes.js"></script>
   </head>
   <body>
-    <h1>hello there</h1>
-    <p id="quote"></p>
-    <p id="author"></p>
-    <button type="button" id="new-quote">New quote</button>
-  </body>
+ <main>
+  <div class="container-quotes">
+    <h1>DAILY INSPIRATION</h1>
+      <div class="container">
+        <div>
+          <p id="quote">"Life isn't about getting and having, it's about giving and being."</p>
+          <p id="author">"Kevin Kruse"</p>
+          <button class="cta" type="button" id="new-quote" id="gradiente">New Quote</button>
```

Comment:
Two `id` attributes in one element?

Reply:
Hi CyYaun,
I was trying to add a background image on my container but didn't work and I forgot it there. 
I just deleted this. 
thank you